codecs/rotate/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "rotate",
+  "scripts": {
+    "build": "mv rotate.{as,ts} && asc rotate.ts -b rotate.wasm --validate --optimize && mv rotate.{ts,as}"
```

Comment:
Nice!. Btw you could get more performant output use `-O3` instead `--optimize`

Reply:
Interesting! Why doesn’t the scaffold of `asinit .` use that?

Reply:
`optimize` flag equivalent to `-O2s` - which mean moderate speed and size. This do most of compilers because `-O3` produce maximal performant output but also slow compilation and not always best binary size.

Reply:
I know `-Os`, `-Oz`, `-O2` and `-O3`, but this is the first time I have heard of `-O2s`. 

Considering we are at ~300 bytes, I pushed for `-O3` ;)